### ISSUE TITLE
010 Library of Congress Control Number

### ISSUE THREAD

**Issue by CECSpecialistI** [uwlib-cams/MARC2RDA#8]
Map the [OXX spreadsheet](https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/0XX.csv)

**Comment by cwarc** [comment]
https://docs.google.com/spreadsheets/d/1O_tBQqCMUpDz4SS_u5SWX0fP9Ww9Szn8iN5L2lquvyE/edit#gid=9363292

Hello Everyone,

Please see the first pass of 010 - Library of Congress Control Number (NR). Is it possible that someone might review my work?

Thank you.

Best,
Alice

**Comment by dchen077** [comment]
@corialanus @sarahruska Would either of you take a look at the test output for 010? Just wanted to make sure I am interpreting the transform note correctly. Thanks! Here is the [link](https://github.com/uwlib-cams/MARC2RDA/blob/main/Working%20Documents/transformationCode/test_output/0xx/010Test-RDA.xml).

**Comment by tmqdeborah** [comment]
Jumping in here, the hash marks (#) indicate spaces, so the output should be [x] spaces, or left anchored letters + spaces to equal [x] characters, before the number AND one or no space after the number:
Library of Congress control number: "   85153773 "
Library of Congress control number: "ms 89001579 "

See the explanation of the structure of the number at the bottom of the page at [010](https://www.loc.gov/marc/bibliographic/bd010.html
)
$a###68004897#
[LCCN structure A; number on printed card: 68-4897]

$a##2001045944
[LCCN structure B; number in print form: 2001-45944]

Basically, map what is provided in the subfield as given in the subfield.
